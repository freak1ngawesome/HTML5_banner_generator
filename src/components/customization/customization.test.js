import React from 'react';
import Customization from './customization';
import renderer from 'react-test-renderer';
import store from '../../store';

describe('Testing <Customization/>', () => {
  const custom = renderer.create(<Customization store={store}/>).toJSON();

  it('Component has rendered correctly',() => {
    expect(custom).toMatchSnapshot();
  });
});
