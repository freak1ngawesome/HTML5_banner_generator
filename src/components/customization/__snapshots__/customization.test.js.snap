// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Customization/> Component has rendered correctly 1`] = `
<section
  className="customization"
>
  <div
    className="customization__block"
  >
    <button
      className="customizer_btn"
      onClick={[Function]}
    >
      show
    </button>
    <form
      className="customization__form"
    >
      <div
        className="customization__form_item banner_sizes"
      >
        <h2
          className="customization__form_label"
        >
          Banner size
        </h2>
        <div
          className="customization__form_inputs"
        >
          <span>
            X:
          </span>
          <input
            className="customization__form_text"
            onChange={[Function]}
            type="text"
            value="282"
          />
          <span>
            Y:
          </span>
          <input
            className="customization__form_text"
            onChange={[Function]}
            type="text"
            value="376"
          />
        </div>
      </div>
      <div
        className="customization__form_item background"
      >
        <h2
          className="customization__form_label"
        >
          Background
        </h2>
        <div
          className="customization__form_inputs"
        >
          <input
            className="customization__form_color"
            onInput={[Function]}
            type="color"
            value="#000000"
          />
          <input
            className="customization__form_text"
            onChange={[Function]}
            type="text"
            value="#000000"
          />
        </div>
      </div>
      <div
        className="customization__form_item font_color"
      >
        <h2
          className="customization__form_label"
        >
          Font color
        </h2>
        <div
          className="customization__form_inputs"
        >
          <input
            className="customization__form_color"
            onInput={[Function]}
            type="color"
            value="#ffffff"
          />
          <input
            className="customization__form_text"
            onChange={[Function]}
            type="text"
            value="#ffffff"
          />
        </div>
      </div>
      <div
        className="customization__form_item font_size"
      >
        <h2
          className="customization__form_label"
        >
          Font size
        </h2>
        <div
          className="customization__form_inputs"
        >
          <input
            className="customization__form_text"
            onChange={[Function]}
            type="text"
            value="44"
          />
        </div>
      </div>
      <div
        className="customization__form_item imgURL"
      >
        <h2
          className="customization__form_label"
        >
          Enter img link or URI
        </h2>
        <div
          className="customization__form_inputs"
        >
          <input
            className="customization__form_text"
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        className="customization__form_item link"
      >
        <h2
          className="customization__form_label"
        >
          Enter link to your site
        </h2>
        <div
          className="customization__form_inputs"
        >
          <input
            className="customization__form_text"
            onChange={[Function]}
            type="text"
            value=""
          />
        </div>
      </div>
      <button
        className="reset_btn"
        onClick={[Function]}
      >
        reset
      </button>
    </form>
  </div>
</section>
`;
